### PR TITLE
[ADD] l10n_kw: kuwait chart of account

### PR DESCRIPTION
**Pull request by niyasraphy**
add kuwait chart of account and banks

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by epictete** [comment]
Thanks for the PR @niyasraphy :+1: 
Could you please have a look @jco-odoo ?

File: l10n_kw/data/res_bank_data.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
```
Comment (jco-odoo) [inline l10n_kw/data/res_bank_data.xml:2]: could be shorter in csv?

**Review by jco-odoo — COMMENTED** [review]
It could be interesting to check a bit further for VAT report, P&L and balance sheet.  (or if you have some reference/doc, so we can check from our side as well)

File: l10n_kw/data/res_bank_data.xml (hunk 1 above)
Comment (niyasraphy) [reply l10n_kw/data/res_bank_data.xml]: @jco-odoo updated with csv

**Pull request closed without merging**